mmdet3d/models/detectors/mink_single_stage.py

```diff
@@ -87,3 +87,47 @@ def extract_feat(
         if self.with_neck:
             x = self.neck(x)
         return x
+
+    def _load_from_state_dict(self, state_dict: OrderedDict, prefix: str,
```

Comment:
We should also set _version in this model, otherwise the newly trained and saved checkpoint will also use the following logic to load checkpoints